00-RELEASENOTES

```diff
@@ -1,16 +1,154 @@
+* Fix CLUSTER SLOTS/NODES showing wrong port after updating port/tls-port by @enjoy-binbin (#2186)
+* CLIENT UNBLOCK should't be able to unpause paused clients by @enjoy-binbin (#2117)
+* Fix replica can't finish failover when config epoch is outdated by @enjoy-binbin (#2178)
+* CVE-2025-32023 prevent out-of-bounds write during hyperloglog operations (#2146)
+* CVE-2025-48367 retry accept on transient errors (#2315)
+* Allow dynamic modification of io-threads num by @ayush933 (#2033)
+* Correctly handle large cluster bus extensions which may have resulted in dropped cluster packets by @madolson (#2144)
+* Improve clarity of errors for GEO commands when member does not exist by @chx9 (#1943)
+* Detect SSL_new() returning NULL in outgoing connections by @zuiderkwast (#2140)
+* Only mark the client reprocessing flag when unblocked on keys by @ranshid (#2109)
+* Fix memory corruption in sharded pubsub unsubscribe by @uriyage (#2137)
+* Respect process umask when creating data files by @kronwerk (#1725)
+* Allow mixing quoted and unquoted inline args by @Fusl (#2098)
+* Disallow sending REPLY ON / OFF / SKIP inside a multi-exec transaction by @sarthakaggarwal97 (#1966)
+* Environment variable REDISCLI_AUTH to VALKEYCLI_AUTH conversion by @HiranmoyChowdhury (#1995)
+* Avoid shard id update of replica if it doesn't match with primary shard id by @hpatro (#573)
+* Fix engine crash on module client blocking during keyspace events by @yairgott (#1819)
+* Fix incorrect lag reported in XINFO GROUPS by @nesty92 (#1952)
+* Ignore stale gossip packets that arrive out of order by @enjoy-binbin (#1777)
+* Fix panic in primary when blocking shutdown after previous block with timeout by @murphyjacob4 (#1948)
+* fix cluster slot stats assertion during promotion of replica by @Fusl (#1950)
+* Fix crash during TLS handshake with I/O threads by @uriyage (#1955)
+* Change "Redis ver." to "Valkey ver." in LOLWUT output by @sarthakaggarwal97 (#1559)
+* Fix RANDOMKEY infinite loop during CLIENT PAUSE by @li-benson (#1850)
+* Fix ACL LOAD crash on replica since the primary client don't has a user by @bogdanp05 (#1842)
+* Save config file and brocast the PONG when configEpoch changed by @enjoy-binbin (#1813)
+* Fix bug where invalidation messages were getting sent to closing clients by @madolson (#1823)
+* cmd's out bytes need count deferred reply by @soloestoy (#1760)
+* valkey-cli: ensure output ends with a newline if missing when printing reply by @xbasel (#1782)
+* Remove unicode optimization in Lua cjson library by @rjd15372 (#1785)
+* Fix temp file leak druing replication error handling by @enjoy-binbin (#1721)
+* Enable TCP_NODELAY by default in incoming and outgoing connections by @sungming2 (#1763)
+* Move TCP/TLS specific options from generic client to connection type by @sungming2 (#1706)
+* Fix raxRemove crash at memcpy() due to key size exceeds max Rax size by @VoletiRam (#1722)
+
+### Build and Tooling
```

Comment:
So https://github.com/valkey-io/valkey/pull/2430 is also fixing an issue with benchmark + an issue with RDMA multiple ack.

Either add it here or better in bug-fixes?